lua/orgmode/utils.lua

```diff
@@ -370,4 +370,32 @@ function utils.prompt_autocomplete(arg_lead, list, split_chars)
   end, matches)
 end
 
+function utils.choose(items)
+  items = items or {}
+
+  local output = {}
+  for _, item in ipairs(items) do
+    table.insert(output, { string.format('[%s] ', item.choice_text) })
+
+    if item.desc_text and item.desc_hl then
```

Comment:
I don't think check for `desc_hl` is needed because it defauts to Normal inside the block.

Reply:
Good call.